runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutor.java

```diff
@@ -100,7 +100,7 @@ public PythonEvaluationResult eval(String expression, String prepareEnvironmentS
         try {
             String pythonPath = checkPythonPath();
             tempEvalEnvironment = generateTempResourcesForEval();
-            String payload = generatePayloadForEval(expression, prepareEnvironmentScript, context);
+            String payload = generatePayloadForEval(expression.replaceAll("[\\n\\r]", ""), prepareEnvironmentScript, context);
```

Comment:
Also, is this unix/windows friendly ? `\n\r` is a windows stuff.

Reply:
Yes, we have:

Windows: '\r\n'
Unix/Linux: '\n'

It will replace all '\n' or '\r' characters in the string.